lib/templates/erb/scaffold/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<%# frozen_string_literal: true %>
```

Comment:
was this added by simple form? Or does this overwrite some simpleform functionality? Maybe add a comment to this file describing what it's all about.